.github/instructions/markdown.instructions.md

```diff
@@ -66,7 +66,7 @@ Follow these guidelines for formatting and structuring your markdown content:
 
 ### Abbreviations
 
-- On first use, spell out the term followed by the abbreviation in parentheses; use the abbreviation alone on subsequent uses within the chapter.
+- On first use, spell out the term followed by the abbreviation in parentheses; use the acronym alone on subsequent uses within the chapter.
```

Comment:
Not every abbreviation is an acronym, so we can keep “abbreviation”. This is more accurate.

```suggestion
- On first use, spell out the term followed by the abbreviation in parentheses; use the abbreviation alone on subsequent uses within the chapter.
```